fdbserver/DeltaTree.h

```diff
@@ -25,13 +25,27 @@
 #include "fdbclient/FDBTypes.h"
 #include "fdbserver/Knobs.h"
 #include <string.h>
+#include <emmintrin.h>
```

Comment:
This needs to be guarded for non-Intel platforms, e.g. aarch64

Reply:
Yeah; the #if __AVX__ or #if __SSE4_2__ should be sufficient to let it build on non-intel.